wildfly-standalone-centos7/azuredeploy.json

```diff
@@ -0,0 +1,283 @@
+      "metadata": {
+        "description": "Administrator password for Linux VM user account"
+      }
+    },
+    "dnsLabelPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "DNS Label for the Public IP. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+      }
+    },
+    "wildflyUserName": {
+      "type": "string",
+      "metadata": {
+        "description": "User name for WildFly manager"
+      }
+    },
+    "wildflyPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for WildFly manager"
+      }
+    },
+    "sshPassPhrase": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Pass phrase for SSH certificate"
+      }
+    },
+    "_artifactsLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
+      },
+      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/wildfly-standalone-centos7/"
+    },
+    "_artifactsLocationSasToken": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated."
+      },
+      "defaultValue": ""
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    }
+  },
+  "variables": {
+    "imagePublisher": "OpenLogic",
+    "imageOffer": "CentOS",
+    "imageSKU": "7.5",
+    "baseName": "[uniquestring(parameters('dnsLabelPrefix'),resourceGroup().id)]",
+    "nicName": "[concat('myNic8',variables('baseName'))]",
+    "addressPrefix": "10.0.0.0/16",
+    "subnet1Name": "Subnet-1",
+    "subnet1Prefix": "10.0.0.0/24",
+    "publicIPAddressName": "[concat('myIP8',variables('baseName'))]",
+    "publicIPAddressType": "Dynamic",
+    "vmName": "[parameters('dnsLabelPrefix')]",
+    "vmSize": "Standard_F1",
+    "virtualNetworkName": "MyVNET8",
+    "frontEndNSGName": "[concat(parameters('dnsLabelPrefix'),'-nsg8')]",
+    "storageAccountType": "Standard_LRS",
+    "storageAccountName": "[concat('vsts8',variables('baseName'))]",
+    "ScriptFolder": "scripts",
+    "ScriptFileName": "wildfly-setup-redhat.sh",
+    "subnet1Ref": "[resourceId('Microsoft.Network/virtualNetworks/subnets', variables('virtualNetworkName'), variables('subnet1Name'))]"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Storage/storageAccounts",
+      "name": "[variables('StorageAccountName')]",
+      "apiVersion": "2016-01-01",
```

Comment:
update apiVersion

Reply:
Which version?

Reply:
See the BP guidelines for the repo

Reply:
There is no specific version for this. Let me know the exact version that you need to update.

Reply:
Read the contribution guide: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#api-versions

Reply:
@bmoore-msft According to the the link, I can use the latest one or no more than 12 months old. Too vague, I need a certain version for this.